server/pension/tests.py

```diff
@@ -1,3 +1,7 @@
 from django.test import TestCase
 
-# Create your tests here.
+class TestPension(TestCase):
+
+    def test_index_view(self):
```

Comment:
@ranmoshe 
Since we're not testing the _index_ anymore then please rename the function, maybe something like `test_api_root_endpoint`